src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends FigureColor {
```

Comment:
The class 'IsoscelesTrapezoid' should not extend 'FigureColor'. Instead, it should implement an interface or extend a class that properly represents its behavior or properties.